nshutijonathan/Banka
# Challenge3 feedbacks

**#### What does this PR do?** 
-Restructure the project
**#### description of tasks to be completed**
-Removed API v1 controllers
-Removed API v1 routes
--Removed API v1 models
**#### How should this be manually tested?** 
clone the repo then navigate in server folder using cmd then
 npm run dev(this command runs the server )after open POSTMAN then localhost:3000 to check the server
#### Any background context you want to provide? 
N/A
#### What are the relevant pivotal tracker stories? 
Story_id:
Pivotal tracker:https://www.pivotaltracker.com/n/projects/2322044
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
